### DISCUSSION TITLE
”从 Valine 迁移至 Waline MySQL/PostgreSQL/SQLite 存储服务“转换工具似乎没法用？

### DISCUSSION THREAD

**Discussion by JesseJeson** [walinejs/waline#784] [Q&A] [answered]
https://waline.js.org/migration/tool.html

**Comment by lizheming** [comment, marked as answer]
有问题的话可以说下具体有什么问题，提供下数据我试一下

**Comment by ceeim** [comment]
我尝试了貌似也是没用，点击转换后无任何变化，也许文档需要更新。

**Reply by lizheming** [reply]
也可以先用 LeanCloud 部署上，直接用后台的导入导出功能

**Reply by ceeim** [reply]
> 也可以先用 LeanCloud 部署上，直接用后台的导入导出功能

我现在是 LeanCloud 部署的 Valine 想切换到 Waline，导出数据转换到 MySQL 无变化，数据转换工具不可用。
因为想自建，LeanCloud 目前国内节点需要接入备案比较麻烦。
